wandb/agents/pyagent.py

```diff
@@ -70,6 +70,11 @@ class Agent(object):
 
     FLAPPING_MAX_SECONDS = 60
     FLAPPING_MAX_FAILURES = 3
+    AGENT_MAX_INITIAL_FAILURES = (
+        int(os.getenv(wandb.env.AGENT_MAX_INITIAL_FAILURES, 5))
+        if isinstance(os.getenv(wandb.env.AGENT_MAX_INITIAL_FAILURES, 5), int)
```

Comment:
os.getenv always returns a string or None

For now you might want to just create a function in: env.py like get_agent_report_interval()